omnigibson/robots/active_camera_robot.py

```diff
@@ -114,7 +115,7 @@ def _default_controller_config(self):
 
         return cfg
 
-    @property
+    @cached_property
```

Comment:
Does this do anything really? Like will it cache the child's implementation of this method?

Reply:
it's effectively a lookup within itself so the parent and child should each have their own cache AFAIK